File: TransCoder_tokenized_test_set_functions/java/DIVIDE_LARGE_NUMBER_REPRESENTED_STRING.java
static String longDivision ( String number, int divisor ) {
  String ans = "";
  int idx = 0;
  char [ ] num = number . toCharArray ( );
  int temp = num [ idx ] - '0';
  while ( temp < divisor ) temp = temp * 10 + ( num [ ++ idx ] - '0' );
  idx += 1;
  while ( num . length > idx ) {
    ans += ( temp / divisor );
    temp = ( temp % divisor ) * 10 + num [ idx ++ ] - '0';
  }
  if ( ans . length ( ) == 0 ) return "0";
  return ans;
}

File: results/java/DIVIDE_LARGE_NUMBER_REPRESENTED_STRING.cpp
string f_filled ( string number , int divisor ) {
  string ans ;
  int idx = 0 ;
  string num = number ;
  int temp = num [ idx ] - '0' ;
  while ( temp < divisor ) {
    temp = temp * 10 + ( num [ ++ idx ] - '0' ) ;
  }
  idx += 1 ;
  while ( num . length ( ) > idx ) {
    ans += ( temp / divisor ) ;
    temp = ( temp % divisor ) * 10 + num [ idx ++ ] - '0' ;
  }
  if ( ans . length ( ) == 0 ) return "0" ;
  return ans ;
}
